### PR TITLE
qa/deepsea: separate ses_qa.py task for non-DS testing and implement smoke test for influx MGR plugin

### PR DESCRIPTION
**Pull request by smithfarm**
Many of the test cases we want to implement have nothing to do
with DeepSea except that DS is used to deploy the Ceph cluster.

Implement a separate task for these test cases, and reserve
deepsea.py as much as possible for DS-specific code.

Move the MGR plugins test cases into this new task (this includes
implementing a new test case for the influx plugin).

Signed-off-by: Nathan Cutler <ncutler@suse.com>

### CODE REVIEW

File: qa/tasks/scripts.py [hunk 1]
```diff
@@ -0,0 +1,394 @@
+from util import (
+    remote_run_script_as_root,
+    )
+
+proposals_dir = "/srv/pillar/ceph/proposals"
+
+
+class Scripts:
+
+    script_dict = {
```
Comment (kshtsk) [inline qa/tasks/scripts.py]: I suggest to get rid of this dictionary, in store scripts close to the scripts.py file, for example, directory structure can look like:
```
qa/tasks/scripts.py
qa/tasks/scripts/ceph_cluster_status.sh
a/tasks/scripts/ceph_version_sanity.sh.sh 
qa/tasks/scripts/create_all_pools_at_once.sh.sh 
qa/tasks/scripts/custom_storage_profile.sh.sh 
qa/tasks/scripts/mgr_dashboard_module_smoke.sh.sh 
qa/tasks/scripts/mgr_plugin_influx.sh.sh 
qa/tasks/scripts/proposals_remove_storage_only_node.sh.sh 
qa/tasks/scripts/rados_write_test.sh.sh 
qa/tasks/scripts/remove_storage_only_node.sh.sh 
qa/tasks/scripts/rgw_init.sh.sh 
qa/tasks/scripts/rgw_init_ssl.ssh.sh 
qa/tasks/scripts/salt_api_test.sh.sh
```
Correspondingly the function *remote_run_script_as_root* should be part of Script class and look like:
```
def remote_run_script_as_root(remote, name):
     local_path = os.path.dirname(os.path.realpath(__file__)) + '/scripts/' + name
     .
     .
     .
``` 
The benefits are:
1) Better coding style.
2) Easier to test run the scripts separately from the python code.

File: qa/tasks/scripts.py [hunk 2]
```diff
@@ -0,0 +1,394 @@
...
+    head -n1 | grep ceph_daemon_stats
+echo "OK" >/dev/null
+""",
+        }
+
+    def __init__(self, master_remote, logger):
+        self.log = logger
+        self.master_remote = master_remote
+
+    def ceph_cluster_status(self, *args, **kwargs):
```
Comment (kshtsk) [inline qa/tasks/scripts.py]: All those functions is an overkill to have. Why you need proxy method for all of them and just don't use remote_run_script_as_root directly?

File: qa/tasks/scripts.py (hunk 1 above)
Comment (smithfarm) [reply qa/tasks/scripts.py]: Good idea. Created https://trello.com/c/sTIp68ze/275-ses6-optimize-scriptspy to track it.

File: qa/tasks/scripts.py (hunk 2 above)
Comment (smithfarm) [reply qa/tasks/scripts.py]: I'm going to get rid of these functions soon.

**Pull request merged**